src/example-types/metricGroups/pivot.js

```diff
@@ -31,15 +32,97 @@ let aggsForValues = (node, schema) =>
     }))
   )(node)
 
-let maybeWrapWithFilterAgg = ({ query, filters, skipFilters, aggName }) =>
-  _.isEmpty(filters) && _.isEmpty(skipFilters)
+let getKey = x => x.keyAsString || x.key
+
+// Similar to Tree.lookup but path is a drilldown which uses keyAsString or key
+let resultsForDrilldown = (type, path, results) => {
+  if (_.isEmpty(path) || !results) return results
+
+  let key = _.first(path)
+  let groups = _.get(type, results)
+  let match = _.find(node => getKey(node) === key, groups)
+  return resultsForDrilldown(type, path.slice(1), match)
+}
+
+let someNotEmpty = _.some(_.negate(_.isEmpty))
```

Comment:
I think it's a little easier to understand if this was called `someValues` or something like that so you don't even need to think about the negation

Reply:
Got deleted after refactor